data/json/npcs/talk_tags.json

```diff
@@ -26,7 +28,10 @@
       "Perfect, those pink tablets will keep us going, take some!",
       "You know what would make all this more bearable?  Pink Tablets!",
       "An odd pink tablet from the ground?  I don't see why not!",
-      "Finally, something to take the edge off."
+      "Finally, something to take the edge off.",
+      "Oh <swear!>, where did you find these?  How many can you fit in your mouth?!",
+      "But nothing compares to those blue and yellow purple pills..",
+      "No better way to escape the Cataclysm than floating through space!"
```

Comment:
Same thing as thorazne, if this doesn't mention pink tablet it's going to very confusing.

Reply:
I can understand why this might be confusing, I can review each of the hallucinatory comments to ensure they explicitly mention the actual drug to make it a bit easier to understand!